tests/sles4sap/ipaddr2/cluster_create.pm

```diff
@@ -0,0 +1,48 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Create cluster
```

Comment:
How this new test module is different from the deploy.pm one? Is this one used in any schedule?

Reply:
`cluster_create.pm` is a part of `configure.pm` before.  I split the old `configure.pm` into two: one is to configure the ssh related operation between test machine; one (`cluster_create.pm`) is to create cluster. Because `registration`, `patch_system` need to be done after the ssh is configured. So you could see `cluster_create` is scheduled after `patch_system`.